node/web/api/paymentApiController.js

```diff
@@ -0,0 +1,37 @@
+module.exports.name = 'paymentApiController';
+module.exports.dependencies = ['router', 'exceptions', 'stripe'];
+module.exports.factory = function (router, exceptions, stripe) {
+    'use strict';
+    var chargeCard,
+        confirmPayment,
+        amount = 400;
+
+    router.post('/api/payment/', function (req, res) {
+        chargeCard(req.body.token, req.body.amount, res, function (err) {
```

Comment:
You're letting the client tell you the price. A savvy user can hack the price of their books to pay whatever they want to pay for the books. It would be better to query the books that are in the user's cart, and calculate the price from them. We wouldn't want to trust prices in the user's cart, either, as the price of the book may change over time.